content/tutorials/08-grafana.md

```diff
@@ -0,0 +1,223 @@
+
+::: div image-l
+<svg title='' src='grafana/configure-plugin.png'/>
+:::
+
+6. Select the **Save & Test** button.
+
+## Create a Pixie panel
+
+First, let's create a dashboard:
+
+1. Click the + icon on the left side menu and select **Create Dashboard**.
+
+Next, let's add a panel:
+
+2. In the New Dashboard view, click **Add an empty panel**.
+
+3. In the Edit Panel view, go to the **Query** tab.
+
+4. Configure the query by selecting -- `Pixie Grafana Datasource Plugin` -- from the [data source selector](https://grafana.com/docs/grafana/latest/panels/queries/#data-source-selector).
+
+5. Click the **Save** icon in the top right corner of your screen to save the dashboard.
+
+::: div image-xl
+<svg title='' src='datasource-selector.png'/>
+:::
+
+### Time series graph of HTTP throughput
+
+1. Copy & paste the following query into the Query Editor.
+
+```python
+# Import Pixie's module for querying data.
+import px
+
+# Load data from Pixie's `http_events` table into a Dataframe.
+df = px.DataFrame(table='http_events', start_time=__time_from)
+
+# Bin the 'time_' column using the interval provided by Grafana.
+df.timestamp = px.bin(df.time_, __interval)
+
+# Group data by unique 'timestamp' and count the total number of
+# requests per unique timestamp.
+per_ns_df = df.groupby(['timestamp']).agg(
+        throughput_total=('latency', px.count)
+    )
+
+# Calculate throughput by dividing # of requests by the time interval.
+per_ns_df.request_throughput = per_ns_df.throughput_total / __interval
+
+# Rename 'timestamp' column to 'time_'. The Grafana plugin expects a 'time_'
+# column to display data in a Graph or Time series.
+per_ns_df.time_ = per_ns_df.timestamp
+per_ns_df.request_throughput = per_ns_df.request_throughput * 1e9
+
+# Output select columns of the DataFrame.
+px.display(per_ns_df['time_', 'request_throughput'])
+```
+
+This plugin uses the Pixie Language ([PxL](https://docs.pixielabs.ai/using-pixie/pxl-overview/)) to query telemetry data collected by the Pixie platform.
+
+The above PxL query outputs a table of timeseries data showing overall HTTP request throughput. Request throughput is calculated by counting the number HTTP requests that Pixie automatically traces in your cluster.
+
+This query uses the `__time_from` and `__interval` macros to add dashboard context to the query. See the full [list of macros](reference/plugins/grafana/#macros) supported by this plugin.
+
+2. On the Panel tab, under the **Visualization** drop-down menu, select **Time series**.
+
+3. Add units to the graph: on the **Field** tab, under the **Standard options** drop-down menu, under the **Unit** menu, select **requests/sec**.
+
+You should see something similar to the following:
```

Comment:
You might want to say you should see data plotted over time, and the exact numbers will vary depending on the traffic in your cluster